GDJS/Runtime/runtimeobject.ts

```diff
@@ -486,13 +491,17 @@ namespace gdjs {
       return {
         x: this.x,
         y: this.y,
+        w: this.getWidth(),
+        h: this.getHeight(),
         zo: this.zOrder,
         a: this.angle,
         hid: this.hidden,
         lay: this.layer,
         if: this._instantForces.map((force) => force.getNetworkSyncData()),
         pfx: this._permanentForceX,
         pfy: this._permanentForceY,
+        n: this.name,
```

Comment:
let's add this name, only if `forceSyncEverything`, otherwise don't (just to save a bit of bandwidth...)
so: `n: syncOptions && syncOptions.forceSyncEverything ? this.name : undefined`